js/src/ui/Form/AddressSelect/addressSelect.js

```diff
@@ -534,7 +536,7 @@ class AddressSelect extends Component {
     });
   }
 
-  handleFocus = () => {
+  handleFocus = (e) => {
```

Comment:
Do we need the (unused) variable here? Consider at least a description name - no `e` that can be anything.